remote-trait-object/src/context.rs

```diff
@@ -55,13 +55,38 @@ mod meta_service {
 }
 use meta_service::{MetaService, MetaServiceImpl};
 
+/// A configuration of a `remote-trait-object` context.
 #[derive(Clone, Debug)]
 pub struct Config {
-    /// This will be appended to the names of various threads spawned by RTO, for an easy debug.
+    /// A name that will be appended to the names of various threads spawned by `remote-trait-object`, for an easy debug.
+    ///
+    /// This can be helpful if you handle multiple contexts of `remote-trait-object`.
     pub name: String,
+
+    /// Number of the maximum of concurrent calls.
+    ///
+    /// Value of this doesn't have anything to do with the number of threads that would be spawned.
+    /// Having a large number of this wouldn't charge any cost except really small additional memory allocation.
+    ///
+    /// [`server_threads`]: ./struct.Config.html#field.server_threads
     pub call_slots: usize,
```

Comment:
If this value is not important, we may use a `Vec` to save slots and make it scale automatically(Or give an enough default value).
Providing a not important option makes the system difficult to use.
You don't need to fix it right now. Please consider it.

Reply:
Ok. Let's keep the parameters as raw as possible for now, for easy debug/test at least.
We might provide more high level interface to setup a parameter later